coops-ui/src/components/Dashboard/TodoList.vue

```diff
@@ -109,90 +109,88 @@
-                        class="actions__more-actions__btn px-0"
-                      @click.native.stop  data-test-id="btn-pending-filing-menu">
-                        <v-icon>mdi-menu-down</v-icon>
-                      </v-btn>
-                    </template>
-                    <v-list ref="pending_actions" class="actions__more-actions">
-                      <v-list-item id="btn-cancel-payment" @click="confirmCancelPayment(item)"
-                      data-test-id="btn-cancel-payment">
-                        <v-list-item-title>Cancel Payment</v-list-item-title>
-                      </v-list-item>
-                    </v-list>
-                  </v-menu>
-                </template>
-
-                <v-btn v-else-if="isError(item)"
-                  class="btn-retry-payment"
+              <p class="date-subtitle"
+                v-if="entityFilter(EntityTypes.BCORP) && isConfirmEnabled(item.type, item.status)"
+              >due {{ item.nextArDate }}</p>
+
+              <!-- pre-empt any buttons below -->
+              <template v-if="inProcessFiling !== undefined && inProcessFiling === item.id">
+                <v-btn text loading disabled />
+              </template>
+
+              <template v-else-if="isDraft(item)">
+                <v-btn class="btn-draft-resume"
                   color="primary"
                   :disabled="!item.enabled"
-                  @click.native.stop="doResumePayment(item)"
+                  @click.native.stop="doResumeFiling(item)"
                 >
-                  <span>Retry Payment</span>
+                  <span>Resume</span>
                 </v-btn>
-
-                <template v-else-if="isPaid(item)">
-                  <!-- no action button in this case -->
-                </template>
-
-                <v-btn v-else-if="!isCompleted(item)"
-                  class="btn-file-now"
+                <!-- more DRAFT actions menu -->
+                <v-menu offset-y left>
+                  <template v-slot:activator="{ on }">
+                    <v-btn color="primary" class="actions__more-actions__btn px-0"
+                      v-on="on" id="menu-activator" :disabled="!item.enabled"
+                    >
+                      <v-icon>mdi-menu-down</v-icon>
+                    </v-btn>
+                  </template>
+                  <v-list ref="draft_actions" class="actions__more-actions">
+                    <v-list-item id="btn-delete-draft" @click="confirmDeleteDraft(item)">
+                      <v-list-item-title>Delete Draft</v-list-item-title>
+                    </v-list-item>
+                  </v-list>
+                </v-menu>
+              </template>
+
+                <template v-else-if="isPending(item)">
+                <v-btn class="btn-resume-payment"
                   color="primary"
-                  :disabled="!item.enabled || coaPending || !confirmCheckbox"
```

Comment:
Have you refactored how the button validates there isn't a `coaPending` or the `confirmCheckbox` isn't checked. I just don't see that code in this PR so i'm a little confused.

Reply:
Argh, diff is playing tricks on us. On the Filed Changed page, click on the gear icon and ignore whitespace changes. You should see only 2 lines deleted.

Reply:
Ahhh gotcha. That caught me off guard, i haven't had to check that before. Consider this resolved!